src/openrct2/world/Balloon.h

```diff
@@ -0,0 +1,30 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2020 OpenRCT2 developers
```

Comment:
```suggestion
 * Copyright (c) 2014-2021 OpenRCT2 developers
```
If we are to be picky hahahaha (I won't open issues for this on the whole PR)